source/guides/recipes/dependency-management.md

```diff
@@ -0,0 +1,101 @@
+(dependency-management-niv)=
+# Managing remote sources with niv
```

Comment:
```suggestion
# Automatically managing remote sources with niv
```

This is to address https://github.com/NixOS/nix.dev/pull/782/files#r1380351386, as I assume the suggestion is to emphasize *automatic*.